src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/PedigreeAnnotation.java

```diff
@@ -111,4 +111,14 @@ void validateArguments(Collection<String> founderIds, GATKPath pedigreeFile) {
     public @Nullable GATKPath getPedigreeFile() {
         return pedigreeFile;
     }
+
+    protected boolean contextHasTrioGQs(final VariantContext vc, final Trio trio) {
```

Comment:
Add a descriptive comment to this method.

Reply:
static?

Reply:
done